input/pagecontent/bfdr-content-transitions.md

```diff
@@ -0,0 +1,94 @@
+# Provider Live Birth
+
```

Comment:
Be sure to include (unless I missed it), Input and Coded Race/Ethnicity.  
THis is missing from BFDR, so the currentProfile would be empty.
ALso, include a list of extensions:   PartialDateTime, and PartialDate,